highfidelity/hifi
# Fix for missing alpha on server restart

https://highfidelity.manuscript.com/f/cases/13860

Test plan:
- Run [this](https://gist.githubusercontent.com/SamGondelman/a94287aeb989320be13da085dd4a5cdb/raw/71fc17436c2c307a23e4bdaa08988934a37eed9c/ForwardPrimitives.js).  The second cube should be slightly transparent.
- Restart the server you're in (if you're using the server from this PR, make sure you're in localhost, not on the serverless domain).  When the entities reappear, the second cube should still be slightly transparent.
